tests/data/vrp_ar169sfp.json

```diff
@@ -3509,102 +3509,6 @@
                     "sensor_prev": null,
                     "user_func": null,
                     "state_name": null
-                },
```

Comment:
Same here. This device I have it running and would rather have a problem if temperature is broken :)